lib/test.gi

```diff
@@ -112,6 +112,9 @@ InstallGlobalFunction(RunTests, function(arg)
     if opts.showProgress = true then
       Print("# line ", pos[i], ", input:\n",inp[i]);
     fi;
+    if opts.showProgress = "dots" then
+      Print(".\c");
+    fi;
```

Comment:
Happy to do that instead, there is just enough of a slowdown caused by my assertion code that tests often trip travis's "no output for too long" check.